src/Components/Logout/Logout.tsx

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import { StorageKey, useLocalStorage } from "../LocalStorage/LocalStorage";
+import "./LogoutS.scss";
+
+interface Props {
+    setLoggedIn(successful: boolean): void;
+}
+
+export const Logout = ({ setLoggedIn }: Props) => {
+    const [userInfo, setUserInfo] = useLocalStorage(StorageKey.userInfo, {
+        email: "",
+        password: "",
+    });
+
```

Comment:
should this userInfo live in the parent component b/c both login and logout need it?